neo/utils.py

```diff
@@ -0,0 +1,856 @@
+# -*- coding: utf-8 -*-
+'''
+This module defines multiple utility functions for filtering, creation, slicing,
+etc. of neo.core objects.
+'''
+
+import neo
+import copy
+import warnings
+import inspect
+import numpy as np
+import quantities as pq
+
+
+def get_events(container, properties=None):
```

Comment:
I think it would be more user-friendly to make this `get_events(container, **properties)`, so that it could be used like:
```python
get_events(seg, event_type='trial start')
```

Reply:
Yes, I agree. I changed get_epochs as well.